### ISSUE TITLE
`slidesToScroll="auto"` doesn't allow clicking when at the end of a carousel

### ISSUE THREAD

**Issue by bmueller-sykes** [FormidableLabs/nuka-carousel#725]
React 16.13.1
Nuka Carousel 4.7.1

I first found this problem in Firefox, but now I think it may not be limited to Firefox.

If you have a `slidesToScroll="auto"` set, and then have a carousel configured to show multiple "tiles" at once, then you scroll to the end of the carousel, you cannot click on several of the tiles. If you then change to `slidesToScroll={3}` (or some other number), then when you get to the end of the carousel, you'll see an empty space at the end of the carousel, which of course you cannot click on, but you can click on all of the tiles that are visible. In `auto` mode, if you scroll to the end, then in the example below, `Tile 10` is clickable (e.g.  you see the pointer cursor), but `Tile 9` and `Tile 8` are not clickable (just showing the default cursor). 

Here's a Codesandbox that shows the problem:
https://codesandbox.io/s/gallant-heyrovsky-9j298?file=/src/App.js

...and here's the all the code from that sandbox: 

```
import React from "react";
import Carousel from 'nuka-carousel';
import "./styles.css";

export default function App() {
  return (
    <div className="App">
      <h1>Nuka Carousel "Auto" slides to scroll bug</h1>
      <div>
<Carousel cellSpacing={12} slidesToScroll="auto" slideWidth={`250px`}>
{[1,2,3,4,5,6,7,8,9,10].map((tile, index) => {
  return (
    <div  style = {{border: '1px solid black',
                backgroundColor: 'yellow',
                width: 250,
                height: 100,}}>Tile {tile}</div>
  )
})}
</Carousel>
      </div>
    </div>
  );
}

```

**Comment by bmueller-sykes** [comment]
Nothing on this yet?

**Comment by sarmeyer** [comment]
Hey @bmueller-sykes thanks for submitting this issue - I haven't had a chance to really dig into it yet, but I'll prioritize it. On my initial pass, it looks like we're assigning visible slides incorrectly for this scenario. There has been some other work done around what to do when slidesToShow and total slide count aren't equally divisible and I think this fits in that bucket.

**Comment by bmueller-sykes** [comment]
@sarmeyer Thanks for the reply. Yeah, in addition to the above issue, I've noticed some cases where a tile at the edge of the carousel is sometimes not clickable. It doesn't always happen, but it does appear that the `inert` class can get applied to a tile that is entirely (or mostly) visible. I haven't been able to figure out exactly what the conditions are to make that happen. I'm sure the code around that condition is really gnarly, but if I had to pick, I'd prefer that a tile be clickable if any part of it is visible.

Is the issue that if a tile is not visible, but is "clickable" that users could still click it? I'm wondering about the need to have the `inert` class applied at all.
